hyperbolic_agentkit_core/actions/deploy_validator_node.py

```diff
@@ -0,0 +1,189 @@
+import json
+import os
+import time
+from collections.abc import Callable
+from typing import Optional
+
+import requests
+from pydantic import BaseModel, Field
+
+from hyperbolic_agentkit_core.actions.hyperbolic_action import HyperbolicAction
+from hyperbolic_agentkit_core.actions.ssh_manager import ssh_manager
+from hyperbolic_agentkit_core.actions.utils import get_api_key
+
+DEPLOY_VALIDATOR_PROMPT = """
```

Comment:
This prompt needs to be a maximum of 1024 characters or else it throws errors

Reply:
Do you want me to change it or you will do it on your end?